frontend/.idea/frontend.iml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
```

Comment:
Why these `IDE` files are again commited? Please review your changes first before pushing them to PR.

Reply:
they'll be removed inm the next commit